client/src/main/java/io/split/client/LocalhostSplit.java

```diff
@@ -0,0 +1,20 @@
+package io.split.client;
+
+public class LocalhostSplit {
+    public String treatment;
+    public String config;
+
+    public LocalhostSplit(String treatment, String config) {
+        this.treatment = treatment;
+        this.config = config;
+    }
+
+    public static LocalhostSplit of(String treatment) {
+        return new LocalhostSplit(treatment, null);
```

Comment:
When a configuration is not defined, should we return an empty map instead of null? It would simplify / remove NPEs